### PR TITLE
UnityEngine namespace to avoid collisions

### PR DESCRIPTION
**Pull request by ms-iankudinova**
## Overview
Fixing namespace (UnityEngine.Mesh) clash, when MRTK is used with other packages.

## Changes
- Fixes:  HandJointVisualizer uses UnityEngine.Mesh as a type. To avoid clashing with other types, full namespace was added.

### CODE REVIEW

File: com.microsoft.mrtk.input/Visualizers/HandJointVisualizer/HandJointVisualizer.cs [hunk 1]
```diff
@@ -29,10 +29,10 @@ public class HandJointVisualizer : MonoBehaviour
 
         [SerializeField]
         [Tooltip("Joint visualization mesh.")]
-        private Mesh jointMesh;
+        private UnityEngine.Mesh jointMesh;
```
Comment (AMollis) [inline com.microsoft.mrtk.input/Visualizers/HandJointVisualizer/HandJointVisualizer.cs:32]: I'm curious why this change is needed?   I assume you're hitting conflicts with you product namespace, but what I don't understand is why.   Unless you're adding a reference to your product assemblies, I don't see how com.microsoft.mrtk.input would pick up your product's namespace.

**Review by AMollis — CHANGES_REQUESTED** [review]

**Review by Cameron-Micka — APPROVED** [review]

**Comment by ms-iankudinova** [comment]
We found a reason, why collision happened (our artifact was referenced inside mrtk.input). We applied fix on our side, so there is no need fot this change. I will close this PR
